### ISSUE TITLE
Suggestion/ Question Ozzy Liner

### ISSUE THREAD

**Issue by MrWerBar** [Wabbit0101/mods_hoardercraft#83]
Ozzy liner is worded Single Install and while using Liner snips to remove it the item disappears so if that is a bug i guess fix that? If not can i suggest rewording to Single Use. #

**Comment by Wabbit0101** [comment]
It is worded correctly -- you can INSTALL it once (installation is the effective 'usage' of a liner); once installed the liner protects the player as long as it's installed. You tried to UN-install the liner as in to re-use it with a subsequent reinstallation and that would be TWO uses. Single use might also imply the liner protects you from an extreme temperature once then goes away which is not what it does.
